src/test/java/integ/publish_to_local_queue/PublishToLocalQueueProcess.java

```diff
@@ -0,0 +1,141 @@
+package integ.publish_to_local_queue;
+
+import static integ.publish_to_local_queue.TestPublishToLocalQueueProcess.DEDUP_ID;
+import static integ.publish_to_local_queue.TestPublishToLocalQueueProcess.PAYLOAD_1;
+import static integ.publish_to_local_queue.TestPublishToLocalQueueProcess.PAYLOAD_1_2;
+import static integ.publish_to_local_queue.TestPublishToLocalQueueProcess.PAYLOAD_2;
+import static integ.publish_to_local_queue.TestPublishToLocalQueueProcess.QUEUE_1;
+import static integ.publish_to_local_queue.TestPublishToLocalQueueProcess.QUEUE_2;
+import static integ.publish_to_local_queue.TestPublishToLocalQueueProcess.QUEUE_3;
+
+import com.google.common.collect.ImmutableList;
+import io.xdb.core.communication.Communication;
+import io.xdb.core.encoder.JacksonJsonObjectEncoder;
+import io.xdb.core.process.Process;
+import io.xdb.core.state.AsyncState;
+import io.xdb.core.state.StateDecision;
+import io.xdb.core.state.StateSchema;
+import io.xdb.gen.models.AsyncStateExecuteRequest;
+import io.xdb.gen.models.CommandRequest;
+import io.xdb.gen.models.CommandWaitingType;
+import io.xdb.gen.models.LocalQueueCommand;
+import io.xdb.gen.models.LocalQueueMessage;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PublishToLocalQueueProcess implements Process {
+
+    @Override
+    public StateSchema getStateSchema() {
+        return StateSchema.withStartingState(new PublishToLocalQueueStartingState(), new PublishToLocalQueueState1());
+    }
+}
+
+class PublishToLocalQueueStartingState implements AsyncState<Void> {
+
+    @Override
+    public Class<Void> getInputType() {
+        return Void.class;
+    }
+
+    @Override
+    public CommandRequest waitUntil(final Void input, final Communication communication) {
+        System.out.println("PublishToLocalQueueStartingState.waitUntil: " + input);
+
+        // will be consumed by PublishToLocalQueueState1
+        communication.publishToLocalQueue(QUEUE_2, PAYLOAD_2);
+
+        return new CommandRequest()
```

Comment:
It will be nicer to keep the internal APIs and SDK APIs separate. Even though they may look similar for now. We want to provide the best coding experience as much as we can. 

Exposing the SDK APIs will have the risk of changing/extensing anything will be very difficult. 

For example, I was too lazy to create a new model for `WorkflowOptions` in iWF java sdk, and just use the same one: https://github.com/indeedeng/iwf-java-sdk/blob/d2ae8439d4b7ed996c20303be57f248eb00fe4ef/src/main/java/io/iworkflow/core/WorkflowState.java#L98

And later on, I wanted to extend it to add new feature like this, but it's impossible...I had to do a hack like this: https://github.com/indeedeng/iwf-java-sdk/blob/main/src/main/java/io/iworkflow/core/WorkflowStateOptionsExtension.java

So my learning is to keep them separate as much as possible espeically in the process/state coding experience. 

For command request, the user code looks more natural :
```
return CommandRequest.anyOf(
    LocalQueueCommand.create( queuName),     
    TimerCommand.byDuration( Duration.OfDay(1) ),
)
```